### ISSUE TITLE
creating environment  not showing the progress 

### ISSUE THREAD

**Issue by kalefranz** [conda/conda#5501]
_From @waqasanjum6 on June 5, 2017 20:6_

Miniconda 4.3.21
platform : Linux , windows 
Miniconda 3

**Actual Result :** Running same command on miniconda3 32&64 bit , 32 bit not showing the package progress while 64 bit showing progress 

![screenshot_4](https://cloud.githubusercontent.com/assets/21064071/26800731/cdf46286-4a53-11e7-8d6c-d0622133b174.png)
![screenshot_5](https://cloud.githubusercontent.com/assets/21064071/26800732/cdf73e5c-4a53-11e7-9575-390a46a7385d.png)

Expected Result : It should show the progress on 32 bit as well 

_Copied from original issue: ContinuumIO/anaconda#902_

**Comment by AndreWin** [comment]
There is no progressbar on Windows 8.1x64 too.

**Comment by mcg1969** [comment]
Just encountered this on Linux. It was a big environment, and I thought conda was hung because of it.

**Comment by mcg1969** [comment]
```
               platform : linux-64                                                                                                                  

          conda version : 4.3.14                                                                                                                    

       conda is private : False                                                                                                                     

      conda-env version : 4.3.14                                                                                                                    

    conda-build version : 2.1.7                                                                                                                     

         python version : 2.7.13.final.0                                                                                                            

       requests version : 2.12.4                                                                                                                    
```

**Comment by kalefranz** [comment]
Resolved with #5546 in conda 4.4.
